### DISCUSSION TITLE
反白處中英文相呼應

### DISCUSSION THREAD

**Discussion by Charming0420** [immersive-translate/immersive-translate#984] [Ideas]
### 新功能 | New Feature

1.  反白中文時，就同時自動反白英文部分，更方便做中英對照
2. 反白英文時亦然

### 为什么您想要添加这些功能 | Why you want to add these features

1. 再看長篇文章時，雖然會做分段翻譯，但有時候想對照著看翻譯目前還做不到

### 提议 | Proposal

沒有建議，不了解該如何實現，不過我正在用有道翻譯的網頁版時，也有這個功能，可以參考看看

**Comment by theowenyoung** [comment]
由于翻译服务为第三方服务，插件目前应该是技术上无法实现类似的功能，谢谢！
